subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_df.c

```diff
@@ -388,8 +380,14 @@ void radio_df_ant_switch_pattern_clear(void)
 
 void radio_df_reset(void)
 {
-	radio_df_mode_set(RADIO_DFEMODE_DFEOPMODE_Disabled);
-	radio_df_cte_inline_set_disabled();
+	/* Initialize to NRF_RADIO reset values
+	 * Note: Only registers that turn off the DF feature and those
+	 *       registers whose bits are partially modified across functions
+	 *       are assigned back the power-on reset values.
+	 */
+	NRF_RADIO->DFEMODE = 0x00000000UL;
+	NRF_RADIO->CTEINLINECONF = 0x00002800UL;
```

Comment:
I am not too fond of magic numbers. Consider using a define, or at least a reference to where we can find the power-on reset values

Reply:
Added defines and comment where to find them.